src/Axiom/Set/Map.agda

```diff
@@ -178,7 +178,7 @@ mapˡ-uniq {inj = inj} uniq = λ h h' → case ∈⇔P h ,′ ∈⇔P h' of λ w
   (((_ , b) , refl , Ha) , ((_ , b') , eqb , Hb)) → uniq Ha
     $ subst _ ( sym
               $ ×-≡,≡→≡
-              $ map₁ (inj (from dom∈ (b , Ha)) (from dom∈ (b' , Hb)))
+              $ map₁ (inj (to dom∈ (b , Ha)) (to dom∈ (b' , Hb)))
```

Comment:
Reverse order of equivalence in definition of `dom∈` to be more consistent with other definitions and so that it's more natural in the sense that `to dom∈` takes a decomposition *to* a dom inclusion.